src/util/useSortTableData.tsx

```diff
@@ -13,10 +13,11 @@ const useSortTableData = (props: Props) => {
   const [sort, setSort] = useState<string | null | undefined>(defaultSort);
   const [sortDirection, setSortDirection] = useState(defaultSortDirection);
 
-  if (sort) {
+  const sortToUse = sort || defaultSort;
+  if (sortToUse) {
     rows.sort((a, b) => {
-      const aVal = a.sortData ? (a.sortData[sort] as string) : "";
-      const bVal = b.sortData ? (b.sortData[sort] as string) : "";
+      const aVal = a.sortData ? (a.sortData[sortToUse] as string) : "";
+      const bVal = b.sortData ? (b.sortData[sortToUse] as string) : "";
```

Comment:
```suggestion
      const aVal = a.sortData ? (a.sortData[sort || defaultSort] as string) : "";
      const bVal = b.sortData ? (b.sortData[sort || defaultSort] as string) : "";
```
`fooToUse` is not a good naming, maybe this can be inlined for the two use cases? Or intercept the `setSort` calls and set it to `defaultSort` in case it is set empty.